courses/filters.py

```diff
@@ -0,0 +1,34 @@
+import datetime
+
+from rest_framework import filters
+
+is_true = lambda value: value.lower() in ('true', 'yes', '1', 'y')
```

Comment:
Pourquoi définir une fonction lambda alors qu'on peut définir une fonction normale ?

    def is_true(value):
        return value.lower() in ['true', 'yes', '1', 'y']